source/EngineTests/CellConnectionTests.cpp

```diff
@@ -61,7 +61,8 @@ TEST_F(CellConnectionTests, addFirstConnection)
 
 TEST_F(CellConnectionTests, addSecondConnection)
 {
-    auto data = CollectionDescription().addCells({
+    CollectionDescription data;
```

Comment:
Use single chained calls like ` auto data = CollectionDescription().cells({...`
Check also the your other changes.